lua/usip_player.lua

```diff
@@ -2509,20 +2534,26 @@ local function writeAllSips(tPlugin, strBaseComSipData, strBaseAppSipData, tUsip
     fConnected, strMsg = tFlasherHelper.connect_retry(tPlugin)
     if fConnected then
         aAttr = tFlasher.download(tPlugin, flasher_path, nil, true, strSecureOption)
+        iResult = WS_RESULT_OK
+    else
+        iResult = WS_RESULT_ERROR_UNSPECIFIED
     end
 
-    -- check if any of the secure info pages are hidden
-    iResult, strMsg, strCalSipData = verifyInitialMode(tPlugin, aAttr)
-
     if iResult == WS_RESULT_OK then
-        -- Set the SIP protection cookie if requested.
-        if fSetSipProtectionCookie then
-            strComSipData = tUsipGen:setSipProtectionCookie(strComSipData)
-        end
+        -- check if any of the secure info pages are hidden
+        iResult, strMsg = verifyInitialMode(tPlugin, aAttr)
+    end
 
-        strAppSipData = apply_temp_diode_data(strAppSipData, strCalSipData)
-        if tUsipConfigDict ~= nil then
-            strComSipData, strAppSipData = tUsipGen.apply_usip_data(strComSipData, strAppSipData, tUsipConfigDict)
+    if iResult == WS_RESULT_OK then
```

Comment:
The temp diode information is no longer copied to the APP SIP. Is this now done in the flasher binary, or no longer necessary?